packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -274,9 +283,22 @@ export default (async function PgIntrospectionPlugin(
       "id"
     );
 
+
+    /**
+     *
+     *
+     * @param array
+     * @param newAttr
+     * @param lookupAttr
+     * @param lookup
+     * @param missingOk
+     */
     const relate = (array, newAttr, lookupAttr, lookup, missingOk = false) => {
```

Comment:
This goes through the objects in `array` and adds a new property `newAttr` to each object, where the value is the result of looking for the content(s) of the objects property `lookupAttr` in the `lookup` hash. Note this works for both scalar and array values of `lookupAttr`. Basically it rehydrates the relations.